app/components/dashboard-myreports.js

```diff
@@ -45,34 +77,34 @@ export default Component.extend(ReportTitleMixin, {
       });
     });
   }),
-  reportResultsList: computed('selectedReport', 'selectedYear', function(){
-    const report = this.selectedReport;
+
+  reportResultsList: computed('selectedReport', 'selectedYear', async function() {
+    const report = await this.selectedReport;
```

Comment:
`selectedReport` looks like a value and not a promise, any reason for the `await` here?

Reply:
Probably an error.